lib/screens/profile_selection_screen.dart

```diff
@@ -93,14 +92,27 @@ class _ProfileSelectionScreenState extends State<ProfileSelectionScreen> {
                 ? Center(
                     child: Padding(
                       padding: EdgeInsets.all(50),
-                      child: Text(
-                        "There is no profiles attached to the current user.",
-                        textAlign: TextAlign.center,
-                        style: TextStyle(
-                          color: Color(0xFF54A1EE),
-                          fontSize: 25,
-                          fontWeight: FontWeight.bold,
-                        ),
+                      child: Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          Text(
+                            "There is no profiles attached to the current user.",
```

Comment:
I know its not a change to the code, but maybe good to have this sentence quickly checked by Andy or Tiffany.  I think it should be "There are no".